app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -973,6 +971,7 @@
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
OK, I got this, looks like  a bug in Xcode, cannot use a project setting for code signing and have it go down to targets if it's Developer ID - Xcode thinks they're for the app store.